### DISCUSSION TITLE
Error in stock.js

### DISCUSSION THREAD

**Discussion by clj55** [zauberzeug/nicegui#953] [General]
Every time I run my web app, I get this error: `Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'enabled')`
![image](https://github.com/zauberzeug/nicegui/assets/103747880/23cfd5d8-64b2-4646-b590-d70db5cf9557)

When I trace the source map, it reaches here: 
![image](https://github.com/zauberzeug/nicegui/assets/103747880/99e03db6-cda3-4119-a73c-d5415945e84e)

I don't think it is affecting my program but anyone else having this error too?

**Comment by falkoschindler** [comment]
Can you provide a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example)?
Without seeing any code it's hard to tell what's going on.

**Comment by falkoschindler** [comment]
I'll close this discussion due to inactivity. But feel free to reopen it if you have more information about the bug.
